### ISSUE TITLE
harderoporphyrin

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5868]
**Preferred term label:**
harderoporphyrin
see https://www.gastrojournal.org/article/S0016-5085(19)40883-4/pdf
**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0010473

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/618892

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Label: Increased urine harderoporphyrin level
Def: Increased amount of harderoporphyrin in the urine.PMID: 31085196

**Comment by pnrobinson** [comment]
added
